forui/lib/src/widgets/checkbox.dart

```diff
@@ -137,15 +137,17 @@ class FCheckbox extends FFormField<bool> {
     properties
       ..add(DiagnosticsProperty('style', style))
       ..add(StringProperty('semanticLabel', semanticLabel))
+      ..add(ObjectFlagProperty('value', value))
       ..add(ObjectFlagProperty.has('onChange', onChange))
+      ..add(ObjectFlagProperty('enabled', enabled))
```

Comment:
I think these bool properties should be `FlagProperty`. `ObjectFlagProperty` is mostly used with nullable callbacks.